### ISSUE TITLE
draw_rectangles breaks subplots

### ISSUE THREAD

**Issue by l0calh05t** [nschloe/matplotlib2tikz#126]
When `draw_rectangles` is used with subplots, for example:
```python
import matplotlib.pyplot as plt
import numpy as np
from matplotlib2tikz import save as tikz_save

data = [0, 1, 2, 3]
plt.subplot(1, 2, 1)
plt.plot(data)
plt.subplot(1, 2, 2)
plt.hist(data)

tikz_save('bug.tex', draw_rectangles=True)
plt.show()
```
(`hist` could be replaced with `plot` and the same error would occur, but in that case the solution would simply be to drop `draw_rectangles`)

Produces a tex file with a `draw` referencing `axis` outside of the groupplot:
```latex
% This file was created by matplotlib2tikz v0.5.16.
\begin{tikzpicture}

\draw[draw=white,fill=lightgray] (axis cs:0,0) rectangle (axis cs:1,1);
\begin{groupplot}[group style={group size=2 by 1}]
\nextgroupplot[
[...]
```
This causes a missing `\endcsname` error.

I am running Python 3.5.1 with matplotlib 1.5.1 and matplotlib2tikz 0.5.16

**Comment by l0calh05t** [comment]
After some further trying around, this seems to be an issue with `draw_rectangles` in general, not only when subplots are involved.

**Comment by nschloe** [comment]
Fixed by PR #135.
